### PR TITLE
docs: editorial review of part 2

### PR DESCRIPTION
**Pull request by SirSpidey**
A rather significant review of the Part 2. Check for accuracy. 

FYI, you still have a number of `TODO`s in the lab

Ping me if you have questions or don't understand the changes.

### CODE REVIEW

File: docs/part2/20-custom-module.md [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
+- The customized JSON file [override.tftpl](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/override.tftpl) file.
+
+    This file is a copy of the JSON definition for our custom topology that you created in the previous step of this lab.
+- The [main.tf](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/main.tf) file.
+
+    This file contains the Terraform logic that calls the landing zone module with the correct parameters. The logic is relatively minimal because it mostly delegates the effort to the landing zone module.
+
+    Notice these two settings in the `main.tf` file:
+
+    - The `override_json_string` input variable is set to `true`, which instructs the landing zone module to look for and use the `override.json` file in the same directory.
```
Comment (SirSpidey) [inline docs/part2/20-custom-module.md:21]: To me, it's confusing that we say that the var is set to `true`, yet the value doesn't include that boolean. Should we say more about what we're doing in that templatefile?

File: docs/part2/20-custom-module.md [hunk 2]
```diff
@@ -1,78 +1,79 @@
...
-}
-```
-
-To run the terraform module on your local environment:
-
-1.  Clone the git repository locally with
-    ```bash
+In this step, you run a Terraform module that automates the creation of the custom landing zone topology that you defined in the JSON file in the [previous step](./10-customizing.md).
+
+The objective now is to provide a basic Terraform module that can be easily consumed and that creates the custom VPC topology. You can consider this module as a level of abstraction that simplifies consumption. Consumers of the module do not need to understand the landing zone code or the JSON file that contains the topology definition. They can run a `terraform apply` command with a minimal number of inputs to deploy the fully customized VPC topology in their account.
```
Comment (argeiger) [inline docs/part2/20-custom-module.md:7]: be easily consumed and ~~that~~ creates the custom VPC...

File: docs/part2/20-custom-module.md (hunk 1 above)
Comment (argeiger) [reply docs/part2/20-custom-module.md:21]: Correct the override_json_string is set to the string value of the templated file that will be used to override the default values.

File: docs/part2/20-custom-module.md [hunk 3]
```diff
@@ -1,78 +1,79 @@
...
+
+    This file is a copy of the JSON definition for our custom topology that you created in the previous step of this lab.
+- The [main.tf](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/main.tf) file.
+
+    This file contains the Terraform logic that calls the landing zone module with the correct parameters. The logic is relatively minimal because it mostly delegates the effort to the landing zone module.
+
+    Notice these two settings in the `main.tf` file:
+
+    - The `override_json_string` input variable is set to `true`, which instructs the landing zone module to look for and use the `override.json` file in the same directory.
+    - The `source` is set to the standard VSI pattern and points to the version 4.4.4 (the most recent version at the time that this lab was written).
```
Comment (argeiger) [inline docs/part2/20-custom-module.md:22]: it is on 4.4.7 now

File: docs/part2/20-custom-module.md (hunk 1 above)
Comment (argeiger) [reply docs/part2/20-custom-module.md:21]: Vincent had - 
The input named `override_json_string` takes the full json definition. In this example, the json passed to the module goes through the templatefile function first to 'inject' the prefix. It is done to ensure uniqueness of the resource group names in the account as indicated in step 1.

Comment (SirSpidey) [reply docs/part2/20-custom-module.md:21]: Got it. I added this info about the placeholders from what's published.

It seems that there might be some differences between the source branch I was using and what's published out there. Hope there aren't many.

File: docs/part2/00-objectives.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# Lab 2: Automating deployment, and sharing through a private IBM Cloud Catalog
+# Lab 2: Automating deployment and sharing through a private catalog
 
-Lab 1 shows how you can leverage existing landing-zone automation to deploy most of the topology, and then customize and deploy on top of that infrastructure.
+In lab 1, you deployed most of the landing zone topology, and then customized the infrastructure.
 
-The purpose of lab 2 is to show of the manual steps in lab 1 can be fully automated. Lab 2 will also show how the fully automated custom solution can be shared with other users in your enterprise through the [IBM Cloud Catalog](https://cloud.ibm.com/catalog).
+Now in lab 2, you use automation to accomplish what you did manually in lab 1. Lab 2 also demonstrates how you can share the fully automated custom solution with other users in your enterprise through the [IBM Cloud catalog](https://cloud.ibm.com/catalog).
```
[the remark this answers is not in the document]
Comment (SirSpidey) [reply docs/part2/00-objectives.md:5]: I've used our tech writing style of keeping most things in the present tense. For example, 
> In lab 1, you provision a secure VPC-based topology...

Feel free to change back to future if it makes more sense to the lab usage.

File: docs/part2/10-customizing.md [hunk 5]
```diff
@@ -1,64 +1,45 @@
...
 
+The landing zone module is designed to enable both lightweight and deep customizations of the VPC topology, including all the services that are deployed to make the VPC topology compliant.
 
+In a nutshell, you can customize the topology in two ways:
+
+- By using Terraform input variables.
+
+    The module accepts more than 70 input variables that you can use to tweak the VPC topology. Consider the input variables as "knobs" that you can turn to adjust the topology.
+- By using a JSON file, named `override.json`
+
```
[the remark this answers is not in the document]
Comment (SirSpidey) [reply docs/part2/10-customizing.md:13]: Changed to `By passing a JSON string value to the `override.json` variable.`

**Comment by SirSpidey** [comment]
Pushed some updates @argeiger.

**Pull request merged**